.gitignore

```diff
@@ -6,3 +6,4 @@ playwright-report/
 
 .idea
 .DS_Store
+config.json
```

Comment:
```suggestion
//config.json
```

Reply:
what does the double slash do? I only know a single slash.